pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -147,46 +147,40 @@ func TestHTTPCircuitBreakerThresholds(t *testing.T) {
 
 func TestCommonHttpProtocolOptions(t *testing.T) {
 	cases := []struct {
-		clusterName               string
-		useDownStreamProtocol     bool
-		sniffingEnabledForInbound bool
-		proxyType                 model.NodeType
-		clusters                  int
+		clusterName           string
+		useDownStreamProtocol bool
+		proxyType             model.NodeType
+		clusters              int
 	}{
 		{
-			clusterName:               "outbound|8080||*.example.org",
```

Comment:
nit: sniffingEnabledForInbound -> false tests can be removed?

Reply:
I don't think we have any duplicate coverage now. Just had less coverage before

Reply:
All cases are unique combination of direction, useDownStreamProtocol, and proxyType